README.md

```diff
@@ -1,5 +1,56 @@
-# Galasa
-This is a home for all of the Galasa Java code.
+
+
+# Galasa Homepage
+
+[![Build and publish to github pages](https://github.com/techcobweb/galasa.dev/actions/workflows/build.yaml/badge.svg)](https://github.com/techcobweb/galasa.dev/actions/workflows/build.yaml)
+
+This repository contains the source for the Galasa test automation system homepage.
+
+Read the Galasa documentation here:
+- [The latest released version](https://techcobweb.github.io/galasa.dev/latest/)
+- [under development currently but not released yet](https://techcobweb.github.io/galasa.dev/development/)
```

Comment:
These links lead to a 404. Are they going to be populated with something once a build has ran?